common/djangoapps/philu_commands/tests/helpers.py

```diff
@@ -0,0 +1,83 @@
+"""
+Tests philu_commands helpers
+"""
+from __future__ import unicode_literals
+
+from datetime import datetime, timedelta
+
+import pytz
+from django.db.models import signals
+from factory.django import mute_signals
+from mock import Mock, patch
+
+from philu_commands.helpers import generate_course_structure, has_active_certificate
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class MockModuleStoreTest():
+    """
+    Class to mock modulestore class functions, e.g. bulk_operations
+    """
+    def __init__(self, **kwargs):
+        pass
+
+    def __enter__(self):
+        pass
+
+    def __exit__(self, *args, **kwargs):
+        pass
+
+
+class TestHelpers(ModuleStoreTestCase):
+    """
+    Tests for `philu_commands helpers`
+    """
+
+    @mute_signals(signals.pre_save, signals.post_save)
+    def setUp(self):
+        """
+        This function is responsible for creating courses for every test and mocking the function for tests.
+        """
+        super(TestHelpers, self).setUp()
+        self.course = CourseFactory.create(display_name='test course 1', run='Testing_course_1')
+        self.course.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+
+    @patch('philu_commands.helpers.modulestore')
+    def test_generate_course_structure(self, mock_modulestore):
+        """
+        Test 'Successfully generate course structure'
+        """
+        mock_response = Mock(name='mock module store', **{'bulk_operations.return_value': MockModuleStoreTest(),
+                                                          'get_course.return_value': self.course})
+        mock_modulestore.return_value = mock_response
+        course_structure_data = generate_course_structure(self.course.id)
+        expected_course_object = {
+            'structure': {
+                'blocks': {
+                    u'i4x://org.0/course_0/course/Testing_course_1': {
+                        'block_type': 'course', 'graded': False, 'format': None,
+                        'usage_key': u'i4x://org.0/course_0/course/Testing_course_1',
+                        'children': [], 'display_name': u'test course 1'}},
+                'root': u'i4x://org.0/course_0/course/Testing_course_1'}, 'discussion_id_map': {}
+        }
+        assert course_structure_data == expected_course_object
+
+    def test_has_active_certificate(self):
+        """
+        Test 'has active certificates'
+        """
+        does_not_have_cert = has_active_certificate(self.course)
```

Comment:
I feel that the variable name `does_not_have_cert` is misleading, because if its value is `False`, it leads to a double negation, meaning certificate is present. We can name this variable `has_certificate` just like we have used in line 82. `has_certificate` is a generic variable name that inherently shows that it is meant to be a boolean variable that can be used to store both `True` and `False` values.